deploy/helm/templates/autoupdate/autoupdate-cronjob.yaml

```
@@ -17,6 +17,8 @@ spec:
           imagePullSecrets:
             {{- toYaml .Values.imagePullSecrets | nindent 12 }}
           {{- end }}
+          nodeSelector:
+            kubernetes.io/os: linux
```

Do we need to include also architecture requirements, like we do for other workloads? For example:
```suggestion
          affinity:
            nodeAffinity:
              requiredDuringSchedulingIgnoredDuringExecution:
                nodeSelectorTerms:
                - matchExpressions:
                  - key: kubernetes.io/arch
                    operator: In
                    values:
                    - amd64
                    - arm64
                  - key: kubernetes.io/os
                    operator: In
                    values:
                    - linux
```

---

I'm not sure if it is needed necessarily, but it could save us from future problems